test/wasmex/instance_test.exs

```diff
@@ -195,4 +195,58 @@ defmodule Wasmex.InstanceTest do
       {:ok, %Wasmex.Memory{resource: _}} = Wasmex.Instance.memory(store, instance)
     end
   end
+
+  describe "globals" do
+    setup do
+      wat = File.read!("#{Path.dirname(__ENV__.file)}/../example_wasm_files/globals.wat")
+      {:ok, store} = Wasmex.Store.new()
+      {:ok, module} = Wasmex.Module.compile(store, wat)
+      {:ok, instance} = Wasmex.Instance.new(store, module, %{})
+      %{instance: instance, store: store}
+    end
+
+    test t(&Wasmex.Instance.get_global_value/3), context do
+      store = context[:store]
+      instance = context[:instance]
+
+      assert {:error, "exported global `unknown_global` not found"} =
+               Wasmex.Instance.get_global_value(store, instance, "unknown_global")
+
+      assert {:ok, 42} = Wasmex.Instance.get_global_value(store, instance, "meaning_of_life")
+      assert {:ok, -32} = Wasmex.Instance.get_global_value(store, instance, "count_32")
+      assert {:ok, -64} = Wasmex.Instance.get_global_value(store, instance, "count_64")
```

Comment:
Can you maybe add one more assertion where we test the error when accessing an unsupported type? Like a v128 maybe

Reply:
I’ve added asserts for `externref` and `funcref`. It looks like `v128` cannot be used as globals yet.